mne/report.py

```diff
@@ -769,6 +769,14 @@ def _add_figs_to_section(self, figs, captions, section='custom',
             warnings.warn('Could not import mayavi. Trying to render '
                           '`mayavi.core.scene.Scene` figure instances'
                           ' will throw an error.')
+        try:
+            scale = float(scale)
```

Comment:
I prefer explicit checks when possible 

> On 29 avr. 2015, at 19:40, Eric Larson <notifications@github.com> wrote:
> 
> In mne/report.py:
> 
> > @@ -769,6 +769,14 @@ def _add_figs_to_section(self, figs, captions, section='custom',
> >              warnings.warn('Could not import mayavi. Trying to render '
> >                            '`mayavi.core.scene.Scene` figure instances'
> >                            ' will throw an error.')
> > +        try:
> > +            scale = float(scale)
> it can also be callable or None at this point in the code, do you have a different preferred check?
> 
> —
> Reply to this email directly or view it on GitHub.
>